survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) is exploring the possibility of allowing the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) for the HTML content document format in EPUB 3.4, instead of restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it is in earlier versions of EPUB 3.
+
+This initiative aims to address the diminishing use and maintenance of the XML syntax outside the publishing industry while leveraging the future evolution of HTML. 
+
+The proposal does not change the document formats currently allowed in an EPUB. Since EPUB is already reliant on HTML, this proposal merely removes the restriction to the XML syntax.
+
+PMWG is seeking your valuable feedback. What can you see as potential benefits? Do you anticipate any technical challenges? What would you consider a reasonable implementation timeline?  Your insights will help shape the future of EPUB standards. 
+
+Your responses will be kept anonymous but will be made public to ensure transparency. Providing some context about your tools and workflows will help us better understand and address your feedback.
+
+**Question 1:** What challenges, if any, would your organization face in adding support for HTML syntax in your workflows? Please describe any technical, operational, or resource-related issues you foresee.
```

Comment:
Is the intent to add more questions? This only address challenges in benefits and timelines. Should those be added as new questions? E.g. "What benefits do you see from this change? What drawbacks do you see?" and "How much work would such a change be on your workflows and products?". And maybe and final "Other thoughts" section. Or do we want just a single open ended question?